### PR TITLE
chore(firestore-bigquery-export): migration to Node.js v14

### PR DESCRIPTION
**Pull request by dackers86**
## Checklist

- [x] Update extension yaml resources -> properties to contain runtime: {runtime: nodejs14}
- [x] Ensure compilerOptions are updated to include es2020
- [x] Update firebase-functions dependency to latest
- [x] Update url for pricing in readme to https://firebase.google.com/support/faq#extensions-pricing

### CODE REVIEW

**Comment by dackers86** [comment]
### Testing

Fails to install

image

File: firestore-bigquery-export/functions/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "@types/chai": "^4.1.6",
     "chai": "^4.2.0",
     "firebase-admin": "^8.1.0",
```
Comment (Salakar) [inline firestore-bigquery-export/functions/package.json:20]: I think the Admin SDK needs updating also here and is what's causing the failure

File: firestore-bigquery-export/firestore-bigquery-change-tracker/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "@google-cloud/bigquery": "^4.7.0",
     "firebase-admin": "^7.1.1",
```
Comment (Salakar) [inline firestore-bigquery-export/firestore-bigquery-change-tracker/package.json:26]: I think the Admin SDK needs updating also here and is what's causing the failure

**Review by russellwheatley — APPROVED** [review]

File: firestore-bigquery-export/functions/tsconfig.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "lib": ["esnext.asynciterable"],
     "outDir": "lib",
     "types": ["node", "jest", "chai"],
-    "target": "es2018"
+    "target": "es2019"
```
Comment (jhuleatt) [inline firestore-bigquery-export/functions/tsconfig.json:7]:
```suggestion
    "target": "ES2020"
```

**Review by jhuleatt — APPROVED** [review]

**Pull request merged**